server/channels/api4/remote_cluster_test.go

```diff
@@ -14,6 +14,8 @@ import (
 )
 
 func TestGetRemoteClusters(t *testing.T) {
+	t.Skip("Flaky test")
```

Comment:
This should have a JIRA ticket number attached to it so that we can look back and fix it later.